roles/satellite-clone/tasks/main.yml

```diff
@@ -235,6 +246,7 @@
       clone_candlepin_password: "{{ candlepin_password.stdout }}"
       cacheable: true
 
+  # why do we do this after restore? should be documented here - not obvious
```

Comment:
I have for that commit:
```
Fixes #261 - The candlepin database can be restored from an
earlier sat version, so we can migrate it before running the upgrade
to avoid issues.
```

I'm guessing this was more of an issue in previous versions of Satellite.  Sometimes you get these differences with different z-versions. For example, the backup is from 6.3.1 and we are installing the latest 6.3, 6.3.5

Reply:
I'm guessing this was blocking even getting to the upgrade step, which (to my understanding) does migrate candlepin db.